web/js/users.jsx

```diff
@@ -207,6 +208,64 @@ function BanInfo (props) {
   );
 }
 
+
+var usermetas = function (userId) {
+  var Metas = backbone.Collection.extend({
+    url: utils.apiPath(`/users/${encodeURIComponent(userId)}/usermeta`),
+  });
+
+  return new Metas();
+};
+
+var UserMeta = React.createBackboneClass({ 
+
+  onSave: function () {
+    utils.saveModel(
+      this.getModel(),
+      {value:  this.refs.valueInput.value},
+      {},
+      { success: 'Usermeta Saved',
+        error: 'Failed to update meta' }
+    );
+  },
+
+  render: function Meta () {
+    var _meta = this.getModel();
+
+    return (
+      <div className="list-item">
+        <div className="item-id">{_meta.get('id')}</div>
+        <div className="item-costs">
+          <div className="item-cost">
+             
+            <input
+              type='text' 
+              ref='valueInput'
+              defaultValue={_meta.get('value')}
+            />
+          </div>
+        </div>
+                  <button className="btn btn-xs btn-default" onClick={this.onSave}>
```

Comment:
Indentation issue here too

Reply:
Yup, my bad. I left it to the end so I can check the identationt, but I just forgot it.